src/tools/compiletest/src/common.rs

```diff
@@ -651,6 +657,17 @@ pub enum Endian {
     Big,
 }
 
+fn builtin_cfgs(config: &Config) -> Vec<String> {
```

Comment:
Suggestion: `HashSet<String>` or maybe run a `dedup`? There's a bunch of duplicates when I run that on a nightly rustc, doesn't matter that much for the purpose of this check 🤷 

```rs
panic="abort"
panic="unwind"
proc_macro
relocation_model="dynamic-no-pic"
relocation_model="pic"
relocation_model="pie"
relocation_model="ropi"
relocation_model="ropi-rwpi"
relocation_model="rwpi"
relocation_model="static"
rustfmt
sanitize="address"
sanitize="cfi"
```